examples/motoko_examples/counter/canisters/rust/src/lib.rs

```diff
@@ -0,0 +1,59 @@
+use std::cell::RefCell;
+
+thread_local! {
+    static COUNTER_REF_CELL: RefCell<u128> = RefCell::new(0);
+}
+
+//#region Performance
+#[derive(candid::CandidType, Clone)]
+pub struct PerfResult {
+    wasm_body_only: u64,
+    wasm_including_prelude: u64,
+}
+
+thread_local! {
+    static PERF_RESULT_REF_CELL: RefCell<Option<PerfResult>> = RefCell::default();
```

Comment:
This is just a note to help me to remember to ask you about having multiple thread_local!s

Reply:
I tried it out with Jordan and it seemed to work just fine, so we decided to separate the two so that all the performance counter code could live together. That way we can easily pull it all out in the future if needed.